identity-verification/module_3/3.2 Fun with Rekognition DetectFaces and SearchFacesByImage.ipynb

```diff
@@ -0,0 +1,426 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "64907692",
+   "metadata": {},
+   "source": [
+    "# 3.2 Fun with Rekognition DetectFaces and SearchFacesByImage\n",
+    "----\n",
+    "This is fun but optional lab. This lab shows how you can index faces into a collection then use detect faces and search faces by image to identify several faces found in a single image. To do this we'll create a collection, index several face images into the collection. Then we'll search the faces found in a single image against the collection of known faces in order to identify the faces in the image. \n",
+    "\n",
+    "## Steps \n",
+    "\n",
+    "1. Load packages \n",
+    "2. View existing collections \n",
+    "3. Create a new collection \n",
+    "4. Index faces into the collection \n",
+    "5. Search the collection to find and present faces found in an image\n",
+    "6. Clean up\n",
+    "\n",
+    "This  notebook will guide you through on how to compare all faces detected in an image against your Amazon Rekognition Face Collection. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ec7fb7d7",
+   "metadata": {},
+   "source": [
+    "## Step 1. Load Libraries "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ed871982",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import boto3, json, os, io\n",
+    "from PIL import Image, ImageDraw, ExifTags, ImageColor\n",
+    "from IPython.display import Image as IImage\n",
+    "client=boto3.client('rekognition')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ce603810",
+   "metadata": {},
+   "source": [
+    "## Step 2. View your existing collections"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ca7475f8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def list_collections():\n",
+    "\n",
+    "    max_results=10\n",
+    "    \n",
+    "    #Display all the collections\n",
```

Comment:
```suggestion
```
There is a print statement with exactly the same text. I think this is redundant.